lib/rules/html-closing-bracket-newline.js

```diff
@@ -39,7 +52,11 @@ module.exports = {
         type: 'object',
         properties: {
           singleline: { enum: ['always', 'never'] },
-          multiline: { enum: ['always', 'never'] }
+          multiline: { enum: ['always', 'never'] },
+          selfClosingTag: {
+            singleline: { enum: ['always', 'never'] },
+            multiline: { enum: ['always', 'never'] }
+          }
```

Comment:
This is not a valid schema. I think this should work fine:

```suggestion
          selfClosingTag: {
            type: 'object',
            properties: {
              singleline: { enum: ['always', 'never'] },
              multiline: { enum: ['always', 'never'] }
            },
            additionalProperties: false,
            minProperties: 1
          }
```

Reply:
@FloEdelmann Thank you! Just one question, does `minProperties: 1` force users to specify at least one of the two `singleline` or `multiline` properties? If that's the case, should we instead allow people to not set any specific handling for selfClosingTag if they prefer?